frontend/packages/kubevirt-plugin/integration-tests/v2v/vmware/v2v.vmware.import.incorrect.names.negative.scenario.ts

```diff
@@ -0,0 +1,76 @@
+import { VmwareImportWizard } from '../../tests/models/vmwareImportWizard';
+import { VirtualMachineModel } from '../../../src/models';
+import { vmwareVMConfig } from './v2v.configs';
+import { asyncForEach, click } from '@console/shared/src/test-utils/utils';
+import { wrongValues } from '../../tests/utils/constants/vm';
+import { NetworkInterfaceDialog } from '../../tests/dialogs/networkInterfaceDialog';
+import { clickKebabAction } from '../../views/wizard.view';
+import { KEBAP_ACTION } from '../../tests/utils/constants/common';
+import { browser } from 'protractor';
+import * as view from '../../views/importWizard.view';
+import { DiskDialog } from '../../tests/dialogs/diskDialog';
+
+describe('VMWare Wizard validation, negative tests', () => {
+  const wizard = new VmwareImportWizard();
+
+  beforeEach(async () => {
+    await wizard.openWizard(VirtualMachineModel);
+  });
+
+  afterEach(async () => {
+    await wizard.closeWizard();
+  });
+
+  it('VMWare - Import Wizard shows warning when using incorrect VM name', async () => {
```

Comment:
I am getting this for the vmware imports:

```

1) VMWare Wizard validation, negative tests : VMWare - Import Wizard shows warning when using incorrect VM name
   Error: Exceeded number of attempts
       at VmwareImportWizard.<anonymous> /home/ansy/projects/openshift/console/frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts:203:17
       at Generator.next <anonymous>
       at fulfilled /home/ansy/projects/openshift/console/frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts:5:58
   From: Task: Run it"VMWare - Import Wizard shows warning when using incorrect VM name" in control flow
       at UserContext.<anonymous> /home/ansy/projects/openshift/console/frontend/node_modules/jasminewd2/index.js:94:19
       at /home/ansy/projects/openshift/console/frontend/node_modules/jasminewd2/index.js:64:48
       at ControlFlow.emit /home/ansy/projects/openshift/console/frontend/node_modules/selenium-webdriver/lib/events.js:62:21
       at ControlFlow.shutdown_ /home/ansy/projects/openshift/console/frontend/node_modules/selenium-webdriver/lib/promise.js:2674:10
       at /home/ansy/projects/openshift/console/frontend/node_modules/selenium-webdriver/lib/promise.js:2599:53
   From asynchronous test: 
   Error
       at Suite.<anonymous> /home/ansy/projects/openshift/console/frontend/packages/kubevirt-plugin/integration-tests/v2v/vmware/v2v.vmware.import.incorrect.names.negative.scenario.ts:24:3
       at Object.<anonymous> /home/ansy/projects/openshift/console/frontend/packages/kubevirt-plugin/integration-tests/v2v/vmware/v2v.vmware.import.incorrect.names.negative.scenario.ts:13:1
       at Module._compile internal/modules/cjs/loader.js:1063:30
       at Module.m._compile /home/ansy/projects/openshift/console/frontend/node_modules/ts-node/src/index.ts:400:23
       at Module._extensions..js internal/modules/cjs/loader.js:1092:10
       at Object.require.extensions.<computed> [as .ts] /home/ansy/projects/openshift/console/frontend/node_modules/ts-node/src/index.ts:403:12

```

Reply:
This is floating problem, it is reproducing on different tests from time to time, I have no idea how to reproduce it and what causes it

Reply:
Found the reason, when CNV tried to connect to VMWare, it tries some amount of times and then fail with this exact issue. This error is not a problem of my tests, this problem is caused by connectivity issue between CNV and VMWare